content/developers/developer-patterns/navigating-merklelogs/index.md

```diff
@@ -440,19 +350,14 @@ h=2 1 |  2  |  5  |   9  |  12   |  17   |  21   | <-- massif 'tree line'
     0 |0   1 3   4| 7   8|10   11|15   16|18   19|
 ```
 
-We call the look back nodes the *peak stack*, because it always corresponds to
-the peaks of the earlier sub trees. We don't actually pop things off it ever,
-we just happen to reference it in reverse order of addition when adding new
-leaves.
+We call the look back nodes the *peak stack*, because it always corresponds to the peaks of the earlier sub trees. We don't actually pop things off it ever, we just happen to reference it in reverse order of addition when adding new leaves.
```

Comment:
```suggestion
We call the look-back nodes the *peak stack*, because it always corresponds to the peaks of the earlier sub-trees.
We don't pop things off it ever, we just happen to reference it in reverse order of addition when adding new leaves.
```